Include the length of comments in max_schema_info_width only when the with_comment and with_column_comments option is true.

shibaaaa opened the pull request:
## Description

close: #144 

By including the length of comments in max_schema_info_width only when both the with_comment and with_column_comments options are true, we can avoid creating unnecessary space between the column and data type.

shibaaaa commented:
@drwl 
please check out my PR at your most convenient time.

drwl commented:
Thank you for submitting a PR! It's very much appreciated. Could you add test case or provide an example of the comment length intefering with the annotations?

drwl commented:
@shibaaaa PRはありがとう！もう教えてください。テストとか例お願いします。

今日本語を勉強してるけど、間違いがすみません。

shibaaaa commented:
@drwl 
Thank you for writing in Japanese!
Since I'm also studying English, I'll comment in both English and Japanese.

I have added tests for the method we modified this time, so could I ask for a review, please?

-----
(in Japanese)
日本語で書いてくれてありがとうございます！
私も英語を勉強中なので、英語と日本語の両方でコメントします。

今回修正したメソッドのテストを追加したので、レビューをお願いできますか？

shibaaaa commented on spec/lib/annotate_rb/model_annotator/model_wrapper_spec.rb:
```diff
@@ -26,4 +26,78 @@
+        it "should return the max width ncluding the length of the comments and the length of the rdoc" do
+          is_expected.to eq(expect_length)
+        end
+
```
oops, I will remove the unnecessary blank lines.

shibaaaa commented:
@drwl 
I have fixed the Rubocop errors, could I possibly get another review?

drwl commented:
For any future reader, the change looks like this:

Before (with `--without-column-comments`):
```ruby
# == Schema Information
#
# Table name: commented_models(table comment)
#
#  id                             :bigint           not null, primary key
#  comment_col                    :string
#  number_col                     :integer
#  created_at                     :datetime         not null
#  updated_at                     :datetime         not null
#
class CommentedModel < ApplicationRecord
end
```

After (with `--without-column-comments`):
```ruby
# == Schema Information
#
# Table name: commented_models(table comment)
#
#  id          :bigint           not null, primary key
#  comment_col :string
#  number_col  :integer
#  created_at  :datetime         not null
#  updated_at  :datetime         not null
#
class CommentedModel < ApplicationRecord
end
```

drwl commented:
Sorry for the delay, I recently started a new job and haven't been able to spend time to fully review the PR. This looks good to me, thank you for submitting issue and the fix!

お待たせてごめんなさい、最近仕事を始めたので忙しいいになった。PRは良いです。ありがとう！

（この場合、「お待たせてごめんなさい」と「遅れてごめなさい」どちらが良いですか？）

drwl commented:
仕事の後で新し「release」を作る。

shibaaaa commented:
Thank you for review!

「お待たせてごめんなさい」がいいですね👍
